docs/how-to-contribute.md

```diff
@@ -42,8 +42,8 @@ To add a new blog post to the gridsome.org blog:
 
 - Clone [the Gridsome.org repo](https://github.com/gridsome/gridsome.org).
 - Run `yarn` to install all of the website's dependencies.
-- Run `gridsome develop` to preview the blog at `http://localhost:8000/blog`.
-- The content for the blog lives in the `/blog/` folder.
+- Run `gridsome develop` to preview the blog at `http://localhost:8000/blog/`.
+- The content for the blog lives in the `/blog` folder.
```

Comment:
Did you mean to remove this "/"?

Reply:
Yeah, I put it in by mistake and realized after that `/blog` in this context refers to a local folder not a URL, so it shouldn't have the trailing slash.